src/components/TooltipController/TooltipController.tsx

```diff
@@ -235,6 +237,24 @@ const TooltipController = ({
     }
   }, [anchorRefs, providerActiveAnchor, activeAnchor, anchorId, anchorSelect])
 
+  useEffect(() => {
+    if (process.env.NODE_ENV === 'production') {
+      return
+    }
+    if (style?.border) {
+      // eslint-disable-next-line no-console
+      console.warn('[react-tooltip] Do not set `style.border`. Use `border` prop instead.')
+    }
+    if (border) {
+      if (!cssAttrIsValid('border', border)) {
+        // eslint-disable-next-line no-console
+        console.warn(
+          `[react-tooltip] "${border}" is not a valid \`border\`. See https://developer.mozilla.org/en-US/docs/Web/CSS/border`,
+        )
+      }
+    }
+  }, [])
```

Comment:
@danielbarion see how you feel about validating props like this. Planning on doing the same for `opacity`.

Reply:
looks a bit verbose to me, but I don't have a better idea at the moment